integration_tests/tests/apply/new.cy.ts

```diff
@@ -61,7 +70,12 @@ context('Risks', () => {
     // And I click on the start button
     cy.get('[role="button"]').click()
 
-    // Then I see the CRN form
+    // Then I see all existing applications
```

Comment:
minor, but I don't think we need to check this again in this test, as it's not the functionality under test and the circumstances haven't changed from the first one.